plugins/build.gradle.kts

```diff
@@ -3,7 +3,16 @@ plugins {
     id("com.gradle.plugin-publish") version "1.2.0" apply false
 }
 
-class FormaRootConfigurationException(
-    override val message: String,
-    override val cause: Throwable? = null
-) : Exception()
+ext {
+    set("group", "tools.forma")
+    set("version", "0.1.3")
```

Comment:
In the future, need to replace that version configuration on Forma-behavior through target plugin

Reply:
https://github.com/formatools/forma/issues/132